cockroachcloud/quickstart.md

```diff
@@ -92,27 +92,20 @@ postgresql://maxroach:ThisIsNotAGoodPassword@free-tier4.aws-us-west-2.cockroachl
 
 <section class="filter-content" markdown="1" data-scope="console">
 
-<section class="filter-content" markdown="1" data-scope="node">
+<section class="filter-content" markdown="1" data-scope="node go">
 
 {% include cockroachcloud/quickstart/get-connection-string.md %}
 
 </section>
-{% comment %} End node {% endcomment %}
-
-<section class="filter-content" markdown="1" data-scope="go">
-
-{% include cockroachcloud/quickstart/get-connection-string.md %}
-
-</section>
-{% comment %} End go {% endcomment %}
+{% comment %} End node go {% endcomment %}
 
 <section class="filter-content" markdown="1" data-scope="python">
 
 The **Connect to cluster** dialog shows information about how to connect to your cluster.
 
 1. Select **General connection string** from the **Select option** dropdown.
-1. Open a new terminal on your local machine, and run the **CA Cert download command** provided in the **Download CA Cert** section. This certificate is required by most Python clients connecting to {{ site.data.products.db }}.
-1. Open the **General connection string** section, then copy the connection string provided and save it in a secure location.
+2. Open a new terminal on your local machine, and run the **CA Cert download command** provided in the **Download CA Cert** section. This certificate is required by most Python clients connecting to {{ site.data.products.db }}.
+3. Open the **General connection string** section, then copy the connection string provided and save it in a secure location.
```

Comment:
Curious about the reason for renumbering these. We typically use `1.` for all ordered lists, per style guide.

Reply:
This was unintentional. I think maybe VS Code automatically modified these?

Reply:
I will try to add a Vale test to catch this. @nickvigilante